install-panel.sh

```diff
@@ -136,35 +136,16 @@ required_input() {
 }
 
 password_input() {
-  local  __resultvar=$1
-  local  result=''
-
-  while [ -z "$result" ]; do
-    echo -n "* ${2}"
-
-    # modified from https://stackoverflow.com/a/22940001
-    while IFS= read -r -s -n1 char; do
-      [[ -z $char ]] && { printf '\n'; break; } # ENTER pressed; output \n and break.
-      if [[ $char == $'\x7f' ]]; then # backspace was pressed
-          # Only if variable is not empty
-          if [ -n "$result" ]; then
-            # Remove last char from output variable.
-            [[ -n $result ]] && result=${result%?}
-            # Erase '*' to the left.
-            printf '\b \b' 
-          fi
-      else
-        # Add typed char to output variable.
-        result+=$char
-        # Print '*' in its stead.
-        printf '*'
-      fi
+  local result=$1
+  echo -en "* ${2}\n"
+  while [[ -z "$password_main" || ! "$password_main" == "$password_confirm" ]]
+    do
+      read -rsp "Password: " password_main && echo
+      read -rsp "Confirm Password: " password_confirm && echo
+      [[ -z "$password_main" ]] && echo -e "\nError: Password cannot be empty"
+      [[ ! "$password_main" == "$password_confirm" ]] && echo -e "Error: Password do not match\n"
```

Comment:
These prompts do not follow the general styling of the project, i.e. using `*` before text prompts.